mitmproxy/proxy/commands.py

```diff
@@ -39,6 +39,16 @@ def __repr__(self):
         return f"{type(self).__name__}({repr(x)})"
 
 
+class RequestWakeup(Command):
+    """
+    Request a `Wakeup` event after the specified amount of seconds.
+    """
+    threshold: float
```

Comment:
Nit: `threshold` is a bit nondescriptive. How about `delay` (matching https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.call_later)? Or `wakeup_after`?

Reply:
Makes sense. I'll rename it to `delay`.